hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -541,7 +541,11 @@ public TableDescriptorBuilder setReplicationScope(int scope) {
   }
 
   public TableDescriptorBuilder setRegionServerGroup(String group) {
-    desc.setValue(RSGROUP_KEY, new Bytes(Bytes.toBytes(group)));
+    if (group == null) {
```

Comment:
How to unset RSGroup by shell? Add a unit test for shell too?

Reply:
Thank @infraio for reviewing.
I've added an UT to show how to unset by shell, but it's no different than before.